idx/memory/memory.go

```diff
@@ -883,20 +883,27 @@ func (m *MemoryIdx) find(orgId int, pattern string) ([]*Node, error) {
 		}
 	}
 	var startNode *Node
+	var branch string
 	if pos == 0 {
 		//we need to start at the root.
 		log.Debug("memory-idx: starting search at the root node")
 		startNode = tree.Items[""]
 	} else {
-		branch := strings.Join(nodes[0:pos], ".")
+		branch = strings.Join(nodes[0:pos], ".")
```

Comment:
couldn't that just be `nodes[:pos]` without the `0`

Reply:
yep